graviti/dataframe/frame.py

```diff
@@ -303,6 +311,17 @@ def _extend(self, values: "DataFrame") -> None:
         for key, value in self._columns.items():
             value._extend(values[key])  # pylint: disable=protected-access
 
+    def _to_post_data(self) -> List[Dict[str, Any]]:
+        names = self._column_names.copy()
+        values = list(self._columns.values())
+
+        names.append(RECORD_KEY)
+        values.append(self._record_key.to_pyarrow())  # type: ignore[union-attr]
```

Comment:
shall we change the type of `_record_key` to ColumnSeries?

Reply:
a special ColumnSeries without schema?

Reply:
_record_key is string, it can have schema pt.string()